internal/api/websocket.go

```diff
@@ -117,6 +117,8 @@ func handleInput(messageType int, reader io.Reader, err error, buffer chan byte,
 		log.WithError(err).Warn("error while reading WebSocket message")
 		return true
 	}
+
+	log.WithField("messageType", messageType).WithField("message", string(message)).Trace("Received message from client")
```

Comment:
```suggestion
	log.WithField("message", string(message)).Trace("Received message from client")
```

We know that `messageType == websocket.TextMessage` otherwise we would return in line 112.

Reply:
Right, that's true. Thanks!